hack/cluster-clean.sh

```diff
@@ -44,7 +42,9 @@ function main() {
     _kubectl patch cdi cdi --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
 
     set -e
+}
 
+function remove_finalizers() {
```

Comment:
Like `delteResources` consider adding  four more functions: 
`remove_finalizers_from_vmsnapshots`
`remove_finalizers_from_vmsnapshotcontents`
`remove_finalizers_from_vmis`
`remove_finalizers_from_vms`

Reply:
I don't think its needed, we are removing finalizes from the resources, no need imo resolution

Reply:
then maybe in a followup indeed